retrigger/triggerhandler.py

```diff
@@ -367,6 +367,8 @@ async def check_triggers(self, message: discord.Message, edit: bool) -> None:
 
             if trigger.ocr_search and ALLOW_OCR:
                 content += await self.get_image_text(message)
+            if trigger.read_embeds and len(message.embeds) > 0:
+                content += "\n".join(self.convert_embed_to_string(embed) for embed in message.embeds)
```

Comment:
```suggestion
                content += "\n".join(self.convert_embed_to_string(embed, index) for index, embed in enumerate(message.embeds))
```
Required for changes suggested on `convert_embed_to_string`.